roles/openshift_certificate_expiry/library/openshift_cert_expiry.py

```diff
@@ -200,7 +206,7 @@ def __init__(self, subject_string):
         self.subjects = []
         for s in subject_string.split(', '):
             name, _, value = s.partition('=')
-            self.subjects.append((name, value))
+            self.subjects.append((name.encode(), value.encode()))
```

Comment:
I would agree with that in general, but that is not what pyopenssl does. Instead of pyopenssl, we could use cryptography instead, which is already a dependency for pyopenssl. I don't think it explicitly uses bytes internally.